templates/_includes/overview-footer.html

```diff
@@ -1,11 +1,7 @@
 <div class="overview-footer">
   <div class="footer-content">
-    <div class="hxRow">
```

Comment:
(sorry, thought I'd hit submit review earlier)

Reply:
This is a row with a single column, it can be simplified to just the content of the column. I've seen Steven Salinas fall into the same trap. You don't need to use the grid all the time, just when you need to partition a space. Given that normal document flow is columnar, there's no need to apply redundant rows.

Reply:
Ah. Slightly different than other stuff I've worked on. Thanks!